ui/app/components/pending-tx/confirm-send-ether.js

```diff
@@ -27,12 +28,29 @@ function mapStateToProps (state) {
     identities,
     selectedAddress,
     currentCurrency,
+    send,
   }
 }
 
 function mapDispatchToProps (dispatch) {
   return {
-    backToAccountDetail: address => dispatch(actions.backToAccountDetail(address)),
+    clearSend: () => dispatch(actions.clearSend()),
+    editTransaction: txMeta => {
+      const { id, txParams } = txMeta
+      const {
+        gas: gasLimit,
+        gasPrice,
+        to,
+        value: amount,
+      } = txParams
+      dispatch(actions.editTx(id))
+      dispatch(actions.updateGasLimit(gasLimit))
+      dispatch(actions.updateGasPrice(gasPrice))
+      dispatch(actions.updateSendTo(to))
+      dispatch(actions.updateSendAmount(amount))
+      dispatch(actions.updateSendErrors({ to: null, amount: null }))
+      dispatch(actions.showSendPage())
```

Comment:
abstract these into one actions?

Reply:
Done for both send token and send ether in https://github.com/MetaMask/metamask-extension/pull/2557/commits/f034e996c2d4a2c698a9b7a141909a850cbd6129